blockchains/bnb/lib/bnb_transactions_fetcher.js

```diff
@@ -59,7 +60,11 @@ class BNBTransactionsFetcher {
       return nextRange
     }
 
-    this.lastBlockTimestamp = await getLastBlockTimestamp(metrics);
+    this.lastBlockTimestamp = this.bnbTradesMode ?
+    await getLastTradesBlockTimestamp(metrics)
+    :
```

Comment:
Dangling `:` looks strange. Can you pad it a little bit?
And btw, do you use any prettyfier in JS?